src/api/swagger.yml

```diff
@@ -88,6 +88,32 @@ paths:
           description: unexpected error
           schema:
             $ref: "#/definitions/error"
+  /v1/charts/{repo}/{chartName}/versions/{version}/readme:
+    get:
+      operationId: getChartVersionReadme
+      summary: "get Readme.md file for specific chart version"
+      parameters:
+        - name: repo
+          in: path
+          type: string
+          required: true
+        - name: chartName
+          in: path
+          type: string
+          required: true
+        - name: version
+          in: path
+          type: string
+          required: true
+      responses:
+        200:
+          description: Readme.md file content
+          schema:
+            $ref: "#/definitions/resourceDataString"
```

Comment:
New resource to simplify response. It includes `data` root. I need to add `type` and `id` to the payload to make it fully complaint with the jsonapi spec